eras/conway/test-suite/cardano-ledger-conway-test.cabal

```diff
@@ -0,0 +1,71 @@
+cabal-version: 3.0
+
+name:                cardano-ledger-conway-test
+version:             0.1.0.0
+synopsis:            Tests for Cardano ledger conway era
+description:
+  This package builds upon the Alonzo ledger
+bug-reports:         https://github.com/input-output-hk/cardano-ledger/issues
+license:             Apache-2.0
+author:              IOHK Formal Methods Team
+maintainer:          formal.methods@iohk.io
+copyright:           2020 Input Output (Hong Kong) Ltd.
+category:            Network
+build-type:          Simple
+
+data-files:
+  cddl-files/conway.cddl
+  cddl-files/real/crypto.cddl
+  cddl-files/mock/extras.cddl
```

Comment:
These files do not exist yet. We are expecting the conway wire spec to be identical to the babbage wire spec, so we probably do not even want them to exist (except perhaps for consistency). :thinking:

Reply:
So, should these lines be deleted?

Reply:
yes!

Reply:
So they are now deleted